src/main/java/cn/nukkit/command/Command.java

```diff
@@ -211,7 +211,7 @@ public boolean unregister(CommandMap commandMap) {
     }
 
     public boolean allowChangesFrom(CommandMap commandMap) {
-        return commandMap != null && !commandMap.equals(this.commandMap);
+        return this.commandMap == null || this.commandMap.equals(commandMap);
```

Comment:
Do we want to allow changes with null commandMap? Also, was it tested how this affects other methods using allowChangesFrom?